recipes/sigmut/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "sigmut" %}
+{% set version = "1.0" %}
+{% set sha256 = "cf938a2e70bc6dffba86c62fac3028b21e4e4e3bf0cfbd7a251f4f85e67b5f64" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lgpdv/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: generic
+  number: 0
+  skip: True
```

Comment:
This skips all builds.  Are you sure?

Reply:
Thank you for this notice @mariusvniekerk . I've edited the PR but some problems arose related to the linting of the recipe.